src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -34,7 +33,23 @@ class SeekBar extends Slider {
    */
   constructor(player, options) {
     super(player, options);
-    this.update = Fn.throttle(Fn.bind(this, this.update), 50);
+
+    // when playing, let's ensure we smoothly update the play progress bar
+    // via an interval
+    this.updateInterval = null;
+
+    this.on(player, ['play'], () => {
```

Comment:
I think this would probably need to be `playing` as opposed to `play`.

Reply:
Tying to `playing` will result in the same issue we saw before. We can't tie the update flow to these HTML5 video events since they don't trigger enough to allow for smooth play progress.

Reply:
`playing` is only fired once when the video actually starts playing after a `play` and shouldn't make any difference in the update speed, so I agree with @misteroneill that it seems more right to use `playing`. Maybe try it out and if you do see a difference I'd be really interested to know why.

Reply:
Ah actually you guys are completely correct. It seems like`playing` fires only once. No idea why I saw it fire multiple times. I was probably mistaking this for another event. 

https://stackoverflow.com/a/14270041/696130

Seems like I should definitely check for `waiting` too when clearing the interval.